packages/analytics-js/src/components/userSessionManager/utils.ts

```diff
@@ -37,13 +25,15 @@ const generateSessionId = (): number => Date.now();
  * @param {number} sessionId
  * @returns
  */
-const isValidManualSessionId = (sessionId?: number, logger?: ILogger): boolean => {
-  if (!sessionId) return false;
-  if (typeof sessionId !== 'number' || sessionId % 1 !== 0) {
+const isManualSessionIdValid = (sessionId?: number, logger?: ILogger): boolean => {
+  if (!sessionId) {
+    return false;
+  }
+  if (!isNumber(sessionId) || !isPositiveInteger(sessionId)) {
```

Comment:
`isPoisitiveInteger` check should automatically cover `isNumber` check.

Reply:
Addressed